source/extensions/filters/http/well_known_names.h

```diff
@@ -53,7 +53,7 @@ class HttpFilterNameValues {
   // Tap filter
   const std::string Tap = "envoy.filters.http.tap";
   // Wasm filter
-  const std::string Wasm = "envoy.filters.http.wasm";
+  const std::string Wasm = "envoy.wasm";
```

Comment:
Why do we need that?

Reply:
I am just following the lua example.  The fact that it is an http filter is given by context.